### PR TITLE
added sodium from modrinth mavin

### PR DESCRIPTION
**Pull request by MimzyFang**
Probably needs some changes and I probably made some bs but if not I'm happy

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,6 +27,10 @@ reckon {
 
 repositories {
     mavenLocal()
+    maven {
+        name = 'modrinth'
+		url = "https://api.modrinth.com/maven"
+	}
```
Comment (Sollace) [inline build.gradle]: Indendation looks a bit messed up here. Can you fix this before I metge?

Comment (MimzyFang) [reply build.gradle]: sorry for taking so long I was busy

File: build.gradle [hunk 2]
```diff
@@ -27,6 +27,10 @@ reckon {
 
 repositories {
     mavenLocal()
+    maven {
+        name = 'modrinth'
+	url = "https://api.modrinth.com/maven"
+	}
```
Comment (Sollace) [inline build.gradle]: Did you forget the other line? 🤔

Comment (MimzyFang) [reply build.gradle]: No I did not forget I just did not see it

**Review by Sollace — APPROVED** [review]

**Comment by Sollace** [comment]
Had to revert this because the game crashes upon loaded a world with the modrinth version of sodium.

[crashlog.txt](https://github.com/Sollace/Unicopia/files/10170779/crashlog.txt)

**Pull request merged**